crates/runtime/src/instance/allocator/pooling/uffd.rs

```diff
@@ -156,6 +156,7 @@ struct FaultLocator {
     instances_start: usize,
     instance_size: usize,
     max_instances: usize,
+    unguarded_memories_start: usize,
```

Comment:
This name is a little confusing to me (specifically "unguarded"); perhaps simply `mapping_start`?

Reply:
Looking back at this I have no idea what I was thinking